tests/unit/test_dotenvpath.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
+
+import pytest
+
+import ni_measurementlink_service
+from ni_measurementlink_service import _dotenvpath
+
+
+@pytest.mark.parametrize("dotenv_exists", [False, True])
+def test___dotenv_exists_varies___has_dotenv_file___matches_dotenv_exists(
+    dotenv_exists: bool, tmp_path: Path
```

Comment:
How does `tmp_path` get filled in?

Reply:
It's a standard fixture included with pytest. 
https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html